src/plugins/intel_gpu/src/kernel_selector/jitter.cpp

```diff
@@ -1520,8 +1520,23 @@ JitConstants MakeActivationJitConstants(std::vector<kernel_selector::base_activa
     std::string activation_params = "";
     for (size_t i = 0; i < params.size(); i++) {
         std::string activation_suffix = suffix + "_" + toCodeString(i);
-        auto jitConstants = JitConstants{MakeJitConstant("NL_M" + activation_suffix, params[i].m),
-                                         MakeJitConstant("NL_N" + activation_suffix, params[i].n)};
+        std::string nl_m = toCodeString(params[i].m);
+        std::string nl_n = toCodeString(params[i].n);
+        if (params[i].function == ActivationFunction::CLAMP) {
+            if (out_dt == Datatype::INT8) {
+                float max_i8 = 127.f;
+                float min_i8 = -128.f;
+                nl_m = "fmax(" + nl_m + ", " + toCodeString(min_i8) + ")";
```

Comment:
What's the reason to include that in ocl code?
Why not just
```cpp
nl_m = toCodeString(std::max(params[i].m, max_i8));
```
?

Reply:
Fixed to apply clamp with output type range in advance without being included in ocl code as your suggestion. Thanks!